src/ui/scroll.rs

```diff
@@ -74,6 +80,34 @@ where
                 Event::Key(Key::PageDown) if scroller.get_scroller_mut().can_scroll_down() => {
                     scroller.get_scroller_mut().scroll_down(SCROLL_PAGE_DOWN)
                 }
+                Event::Char('G') => scroller.get_scroller_mut().scroll_to_bottom(),
+                Event::Char('g') => {
+                    let now = SystemTime::now();
+                    LAST_TIME_G_WAS_PRESSED.with(|last_time| {
```

Comment:
wow that's really smart! Could you maybe make it so we can use any kind of key combination in the future (so generalize the system a bit more, like vim)? If not that's alright, maybe I can work on it when I have time in the future

Reply:
Thanks! Oooh I think that would be a very neat idea + a fun algorithm to implement; maybe keeping a trie of the keys, and as the keys are pressed, you could traverse the trie and check if any nodes have a registered command. Still unsure how to reason about conflicting/overlapping keybindings (e.g. what if there's `g`, `gg`, and `ggg` bound to different commands). Maybe waiting for no other keypresses to actually execute the command?

I most likely will not have the chance to implement this in the short term. Doing it with just a single key is a lot simpler, _but_ doesn't really scale well as user-configs. I am also happy to revisit this when I have more time in the future!

Reply:
Okay, no problem! Feel free to revisit it in the future but no rush. Thank you for the implementation!